src/Accelerators/NNPA/Conversion/ONNXToZHigh/JsonConfigFile.hpp

```diff
@@ -0,0 +1,52 @@
+#include <functional>
+#include <string>
+
+#include "llvm/Support/JSON.h"
```

Comment:
@chentong319 in our 'onnx-mlir-light' we were trying to reduce dependences on LLVM: is that an issue here? We had Json before for the device placement, so maybe not? Please make sure that this solution is fine with the OM-light.